src/Bridge/Symfony/Routing/RouteNameResolver.php

```diff
@@ -59,6 +60,21 @@ public function getRouteName(string $resourceClass, $operationType /*, array $co
             }
         }
 
+        // Maybe the parent class is a resource
+        foreach ($this->router->getRouteCollection()->all() as $routeName => $route) {
+            $currentResourceClass = $route->getDefault('_api_resource_class');
+            $operation = $route->getDefault(sprintf('_api_%s_operation_name', $operationType));
+            $methods = $route->getMethods();
+
+            if (null !== $operation && (empty($methods) || \in_array('GET', $methods, true)) && null !== $currentResourceClass && is_a($resourceClass, $currentResourceClass, true)) {
```

Comment:
Just asking but maybe we could add this if statement in the previous loop and saved the first value found that would be returned at the after the loop if not null to avoid looping through all routes twice here ?